test/test_parabolic_2d.jl

```diff
@@ -579,8 +579,8 @@ end
     @test_trixi_include(joinpath(examples_dir(), "p4est_2d_dgsem",
                                  "elixir_advection_diffusion_nonperiodic_amr.jl"),
                         tspan=(0.0, 0.01),
-                        l2=[0.007934195641974433],
-                        linf=[0.11030265194954081])
+                        l2=[0.00793370348206051],
+                        linf=[0.11029310107224988])
```

Comment:
Why did this test change? the advection-diffusion equation is in conservation form, isn't it?

Reply:
I need to revert this. They changed because I ran the tests locally on my Mac, but the runners use Ubuntu.